SonarQube.Client/Api/V7_20/GetIssuesRequestWrapper.cs

```diff
@@ -56,6 +57,7 @@ public async Task<SonarQubeIssue[]> InvokeAsync(HttpClient httpClient, Cancellat
             innerRequest.ProjectKey = ProjectKey;
             innerRequest.Statuses = Statuses;
             innerRequest.Logger = Logger;
+            innerRequest.MaxPageNumber = 20; // Retrieve 10000 in 20 pages x 500 items per page;
```

Comment:
The way the client works means if the server version is less than v7.2 this request class won't be used, so the bug will still occur.

The earliest supported `GetIssues` request API is for v5.1. Technically earlier versions of SQ are out of support. However, if there is no additional cost for us in fixing the issue for earlier versions then we might as well do it: it helps customers, and it might also simplify support issues for us in the future (i.e. we won't have to remember that we only fixed this bug for some versions of SQ).